students/ShujinCao/lesson03/assignment/basic_operations.py

```diff
@@ -0,0 +1,59 @@
+"""
+operations
+"""
+import logging
+from customer_model import Customer
+from peewee import fn
+
+def add_customer(customer_id, name, last_name, home_address, phone_number,
+                 email_address, status, credit_limit):
+    try:
+        credit_limit = float(credit_limit)
+    except ValueError as err:
+        logging.error(err)
+        raise
```

Comment:
Yes, it seems peewee isn't checking types. Looks good. Note that since you know what is likely to cause this error ($), you might customize the error message to say they shouldn't use $ or other things that are not numbers.